lib/models/asyncapi.js

```diff
@@ -369,4 +347,50 @@ function assignIdToAnonymousSchemas(doc) {
   schemaDocument(doc, callback);
 }
 
+/**
+ * Recursively go through schema of object type and execute callback.
+ * 
+ * @param {Schema} schema Object type.
+ * @param {Function} callback(schema)
+ *         the function that is called foreach schema found.
+ *         schema {Schema}: the found schema.
+ */
+function recursiveSchemaObject(schema, callback) {
+  if (schema.additionalProperties() !== undefined && typeof schema.additionalProperties() !== 'boolean') {
```

Comment:
`schema.additionalProperties()` could be write to const. I think call 3 times in a row `schema.additionalProperties()` function is unnecessary.

```js
const additionalProperties = schema.additionalProperties();
if (additionalProperties !== undefined && typeof additionalProperties !== 'boolean') {
    recursiveSchema(additionalProperties, callback);
}

// rest of logic identically.
```

Reply:
I did not change the logic when not needed on purpose, I just wanted to get rid of cognitive complexity